pkg/pillar/cmd/zedagent/zedagent.go

```diff
@@ -1805,6 +1840,26 @@ func handleDPCLDelete(ctxArg interface{}, key string, statusArg interface{}) {
 	triggerPublishDevInfo(ctx)
 }
 
+func dpcHasRealChange(dpcList types.DevicePortConfigList) bool {
+	dummy := dpcList
+	for i, l := range dummy.PortConfigList {
+		l.LastFailed = time.Time{}
+		l.LastSucceeded = time.Time{}
+		l.LastError = ""
```

Comment:
Shouldn't we check for updates of error here (and in another places) and send info in case of any?